src/utils.ts

```diff
@@ -1,11 +1,11 @@
-import sha3 = require('js-sha3');
+import { keccak_256 } from '@noble/hashes/sha3';
 
 /**
  * Keccak256 hash
  * @param data The data
  */
 export function keccak256(data: string | ArrayLike<number>): string {
-  return '0x' + sha3.keccak_256(toByteArray(data));
+  return bytesToHex(keccak_256(toByteArray(data)));
```

Comment:
`'0x' + bytesToHex`

Reply:
Bytes to hex already does that

Reply:
https://github.com/joshstevens19/ethereum-bloom-filters/blob/163da5ea767e95362196264b68956b49587a8126/src/utils.ts#L37

Reply:
ok